test/test_cputime.py

```diff
@@ -0,0 +1,64 @@
+from __future__ import annotations
+
+import platform
+import time
+
+from pynisher import CpuTimeoutException, Pynisher, contexts, supports_cputime
+
+import pytest
+
+if not supports_cputime():
+    pytest.skip(
+        f"Can't limit cputime on {platform.platform()}",
+        allow_module_level=True,
+    )
+
+
+def func(execution_time: float) -> tuple[float, int]:
+    """
+    Sleeps for `sleep` second.
```

Comment:
Technically, this doesn't sleep but does an active wait. Not sure if that's intended?

Reply:
For cputime it's intended, we need to keep the process busy to test it. Cputime is not consumed during a `time.sleep()`

Reply:
I should update the docs though